src/state/hooks/WorkspaceHooks.js

```diff
@@ -25,5 +25,5 @@ export const useUserPermissionsOnCurrentWorkspace = () => {
 };
 
 export const useWorkspacesList = () => {
-  return useSelector((state) => state.workspace.list);
+  return useSelector((state) => state.workspace.list?.data);
```

Comment:
Note: this could break other components using this hook (and other branches too). Don't forget to communicate on this change on Slack :smile: